src/Symfony/Component/RateLimiter/Policy/CalendarInterval.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\RateLimiter\Policy;
+
+/**
+ * @author Thibault Gattolliat <contact@thibaultg.info>
+ */
+enum CalendarInterval: string
+{
+    case YEARLY = 'yearly';
```

Comment:
When naming that an interval, I think we should use `year` instead of `yearly` (and same for others)